Examples/MAX78000/SDHC_FTHR/.vscode/settings.json

```diff
@@ -69,10 +69,20 @@
         "${config:MAXIM_PATH}/Libraries/SDHC/ff14/Source"
     ],
     "C_Cpp.default.defines": [
-        
+        "${config:board}",
+        "TARGET=${config:target}",
+        "TARGET_REV=0x4131",
+        "__GNUC__"
     ],
-    "C_Cpp.default.forcedInclude": [
-        "${workspaceFolder}/build/project_defines.h"
-    ]
+    "files.associations": {
```

Comment:
Revert your changes to `.vscode/settings.json`, we don't want to track local modifications